test/cmd/help.sh

```diff
@@ -45,8 +45,8 @@ os::cmd::expect_success_and_text 'oc types' 'Deployment Config'
 os::cmd::expect_success_and_text 'openshift kubectl' 'Kubernetes cluster'
 os::cmd::expect_success_and_text 'oadm' 'Administrative Commands'
 os::cmd::expect_success_and_text 'openshift admin' 'Administrative Commands'
-os::cmd::expect_success_and_text 'oadm' 'Basic Commands:'
-os::cmd::expect_success_and_text 'oadm' 'Install Commands:'
+os::cmd::expect_success_and_text 'oadm' 'Component Installation:'
+os::cmd::expect_success_and_text 'oadm' 'Security and Policy:'
```

Comment:
It's really to verify help works.

On Thu, Jul 7, 2016 at 6:20 AM, Maciej Szulik <notifications@github.com>
wrote:

> In test/cmd/help.sh
> <https://github.com/openshift/origin/pull/9578#discussion_r69883658>:
>
> > @@ -45,8 +45,8 @@ os::cmd::expect_success_and_text 'oc types' 'Deployment Config'
> >  os::cmd::expect_success_and_text 'openshift kubectl' 'Kubernetes cluster'
> >  os::cmd::expect_success_and_text 'oadm' 'Administrative Commands'
> >  os::cmd::expect_success_and_text 'openshift admin' 'Administrative Commands'
> > -os::cmd::expect_success_and_text 'oadm' 'Basic Commands:'
> > -os::cmd::expect_success_and_text 'oadm' 'Install Commands:'
> > +os::cmd::expect_success_and_text 'oadm' 'Component Installation:'
> > +os::cmd::expect_success_and_text 'oadm' 'Security and Policy:'
>
> Shouldn't we have other group names being tested here as well?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/1a1196545cf65401f45500cfc4ad99faae7f1d0c..0c6888358f164488128e0cf7a321892d9170fb4b#r69883658>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p334ZCSn0vQAuqCnZe1JuLzaVEo-ks5qTNL5gaJpZM4I-xTE>
> .
>